src/Mado/QueryBundle/Tests/Dictionary/OperatorsTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Mado\QueryBundle\Tests\Services;
+
+use Mado\QueryBundle\Dictionary\Operators;
+use PHPUnit\Framework\TestCase;
+
+class OpertatorTest extends TestCase
```

Comment:
Operators is a Value Object and is useless test a value. It's behavior is already tested in other tests. If other tests are not present here in this project it could be better to write some tests about methods that uses this VO.